src/orquestra/sdk/_base/_in_process_runtime.py

```diff
@@ -125,12 +143,31 @@ def get_workflow_run_outputs_non_blocking(
     ) -> t.Sequence[t.Any]:
         return self.get_workflow_run_outputs(workflow_run_id)
 
-    def get_available_outputs(self, workflow_run_id: WfRunId) -> t.Dict[str, t.Any]:
-        workflow_run_outputs = self.get_workflow_run_outputs(workflow_run_id)
-        output_task_ids = self._workflow_def_store[workflow_run_id].output_ids
-        return dict(zip(output_task_ids, workflow_run_outputs))
+    def get_available_outputs(
```

Comment:
Issue: The rest of this PR is mostly docstring and interface changes already tested. But this function is completely rewritten. Could you provide at least some basic tests that it works as expected?

Reply:
Yeah sure. TBH I submitted the PR just to see which tests are failing and I'm amazed the whole suite passed.